velox/type/Conversions.h

```diff
@@ -46,6 +46,12 @@ struct SparkCastPolicy {
   static constexpr bool throwOnUnicode = false;
 };
 
+struct SparkTryCastPolicy {
+  static constexpr bool truncate = false;
```

Comment:
How is the truncate flag utilized for the Spark try_cast policy?

Reply:
This truncate flag is used to decide whether an overflow is treated as a failure or is wrapped.